sentinel/transitions/resolve_pending.js

```diff
@@ -36,10 +36,8 @@ var setStateOnTasks = function(tasks, state) {
     var updated = false;
     state = state || 'sent';
     _.each(tasks, function(task) {
-        if (task.state !== state) {
-            utils.setTaskState(task, state);
-            task.timestamp = new Date().toISOString();
-            updated = true;
+        if (utils.setTaskState(task, state)) {
+          updated = true;
```

Comment:
@garethbowen 
This is the only reason I made the function return a value.

Reply:
Yeah, so it's definitely needed. We just need to document exactly what it means in case we want to use the return value somewhere else too.